<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>343</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>556</width>
    <height>343</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>556</width>
    <height>343</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <widget class="QPushButton" name="But_Send2">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string notr="true">'2' - ecnoder postion</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="But_Connect">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>23</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>23</height>
       </size>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="But_Settings">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>23</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>23</height>
       </size>
      </property>
      <property name="text">
       <string>Settings</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QComboBox" name="comboBox">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>23</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>23</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="But_Send1">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>'1' - drive</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="terminal_res.qrc">:/images/redStone.png</pixmap>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="checkBox_N_Enable">
      <property name="text">
       <string>N_Enable</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QCheckBox" name="checkBox_MS1">
      <property name="text">
       <string>MS1</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QCheckBox" name="checkBox_MS2">
      <property name="text">
       <string>MS2</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QCheckBox" name="checkBox_MS3">
      <property name="text">
       <string>MS3</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QCheckBox" name="checkBox_STEP">
      <property name="text">
       <string>STEP</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QCheckBox" name="checkBox_DIR">
      <property name="text">
       <string>DIR</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" rowspan="6" colspan="3">
     <widget class="QTextEdit" name="textEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="terminal_res.qrc"/>
 </resources>
 <connections/>
</ui>
